### ISSUE TITLE
Tags issue with Zerochan

### ISSUE THREAD

**Issue by Deltako806** [Bionus/imgbrd-grabber#2573]
**Bug description**

Some tag can be used on Zerochan website, but not on Grabber search engine.

**Steps to reproduce**

Example :

On Grabber :
1. I search Gokukoku_no_Brynhildr with Zerochan as a source,
2. I have nothing. 

On Zerochan :
1. I search Gokukoku_no_Brynhildr,
2. I get 85 images.

**Expected behavior**

I think, I should get on Grabber the same pictures that are on Zerochan.

**Context**

I don't think I need to add something here.

**Screenshots**

simple research on Grabber: no pictures
![image](https://user-images.githubusercontent.com/96786147/148697899-b56f095d-b729-4335-8ca8-d638eefdd289.png)

same research but on Zerochan: 85 pictures
![image](https://user-images.githubusercontent.com/96786147/148697297-66d17d93-922d-4bcc-b905-676ce96a7b9e.png)

**System information**

I don't think I need to add something here too.

**Additional context**

I tried to understand the problem and I made some discoveries 

first :
I search the artist kuroha_neko on grabber with Zerochan as a source,
![image](https://user-images.githubusercontent.com/96786147/148697526-d2d735c5-a7e3-4516-bcb0-20989e87a8a6.png)

I get 44 pictures and some of them have the tag Gokukoku_no_Brynhildr, 
![image](https://user-images.githubusercontent.com/96786147/148697532-6a4b52fe-f0b7-40a0-a5c1-db2bae6cbe44.png)

I click on this tag (Gokukoku_no_Brynhildr),
I still get nothing on grabber.

second :
If copy lick location :![image](https://user-images.githubusercontent.com/96786147/148697618-b79864c2-7964-495b-b59a-4e40e60d6083.png)

I get this :
https://www.zerochan.net/Gokukoku%20no_Brynhildr?p=1
and I get nothing

but with :
https://www.zerochan.net/Gokukoku+no+Brynhildr?p=1
or :
https://www.zerochan.net/Gokukoku%20no%20Brynhildr?p=1
I have the same good result 

In conclusion, I think that the second "_" in Gokukoku_no_Brynhildr is supposed to be "converted" into "%20"
in order to get the good "Gokukoku%20no%20Brynhildr"
but for some reason, It doesn't 

I think this bug shouldn't be hard to get solved, but I might be wrong

**Comment by Bionus** [comment]
Indeed, it seems that the automatic replacement of underscores by spaces for zerochan was only replacing the first one instead of all of them, so it doesn't work with tags with more than one space 😅 

I just pushed a fix, thanks 👍
